### DISCUSSION TITLE
Section 3: Math Masters. mulWad() function. How was the error selector obtained?

### DISCUSSION THREAD

**Discussion by PavelNaydanov** [Cyfrin/assembly-evm-opcodes-and-formal-verification-course#27] [Q&A]
Hi, guys! 

I see [mulWad()](https://github.com/Cyfrin/2-math-master-audit/blob/main/src/MathMasters.sol#L54) function in MathMasters repo.
The comment syas that `0xbac65e5b` equals `MathMasters__MulWadFailed()`. 

I tried to call `cast sig "MathMasters__MulWadFailed()"` and i got `0xa56044f7`.

What am I getting wrong?

**Comment by PavelNaydanov** [comment]
Oh, i'm sorry. I found this issue before i looked at the relevant lesson where was this error found. The issue has been resolved

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
